src/KingSystem/Event/evtInfoData.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <KingSystem/Resource/resLoadRequest.h>
+#include <KingSystem/Utils/Byaml/Byaml.h>
+#include <heap/seadDisposer.h>
+#include <prim/seadSafeString.h>
+
+namespace ksys::evt {
+
+class InfoDataUnknownType {
+public:
+    virtual ~InfoDataUnknownType();
+};
+
+class InfoData {
+    SEAD_SINGLETON_DISPOSER(InfoData)
+    InfoData();
+    virtual ~InfoData();
+
+public:
+    bool getEntry(al::ByamlIter* iter, const sead::SafeString& eventFlowName,
+                  const sead::SafeString& entryPoint);
+    bool getSceneChangeEventTraverseLimit(const sead::SafeString& eventFlowName);
+    void init(sead::Heap* heap);
+
+private:
+    void doInit(sead::Heap* heap, OverlayArena* a3);
+    al::ByamlIter* rootIter = nullptr;
+    ksys::res::Handle res2;
```

Comment:
This should be called mHandle.

It's called res2 in the IDA database because EventInfoData was reverse engineered in 2018 before I figured out that this was a resource handle class...